specs/host_list_view.adoc

```diff
@@ -0,0 +1,148 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of hosts discovered by Tendrl
+
+Display list of hosts discovered by Tendrl, whether they currently associated with a cluster or not. The list should have functionality for 
+
+* filtering by name, cluster, role and status.
+* sorting by name.
+
+Host list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of hosts discovered by Tendrl along with their details.
+
+The details shown for each host in UI is -
+
+* Host Name
```

Comment:
Does the `Host Name` means just host name (the first part without the domain) or `FQDN`? Especially in the case of `FQDN`, it might be quite long and it might break the rest of the layout.